exotic/exotic.py

```diff
@@ -2303,7 +2379,7 @@ def main():
                                 elif pDict[k] == 0 or np.isnan(pDict[k]):
                                     print(" WARNING! {} uncertainty is 0. Please use a non-zero value in inits.json".format(k))
                                     pDict[k] = 1
-                            elif not pDict[k]:
+                            elif pDict[k] == None:
                                 print(" WARNING! {} is None. Please use a numeric value in inits.json".format(k))
```

Comment:
please give the user the option to add it at the command line--- that way they don't have to go back and restart the code after editing the inits.json file